docs/page_objects.rst

```diff
@@ -103,6 +103,36 @@ elements with nested encapsulation or PageElement identified by other selector t
 
     login_button = Button(By.CSS_SELECTOR, "css_selector", shadowroot="shadowroot_css_selector")
 
+Webview
+~~~~~~~~~~
+
+Page elements have an optional argument *webview*, a boolean that indicates if the page element is in a webview context
+(default value is False). Only apply to mobile tests, where we need to do a change to webview context to find an
+element, which is in a webview. This argument be used only if the configuration property
+*automatic_context_selection* is True.
+
+If *webview* argument is True but webview_context_selection_callback is not defined, then the default webview context
```

Comment:
y aquí " will apply" : then the default webview context change behaviour will apply